pkg/system/phase2_creating.go

```diff
@@ -746,6 +746,15 @@ func (r *Reconciler) ReconcileRootSecret() error {
 func (r *Reconciler) ReconcileDB() error {
 	var err error
 	if r.NooBaa.Spec.DBType == "postgres" {
+		util.KubeCreateSkipExisting(r.PostgresDBConf)
+
+		// this config map is required by the NooBaaPostgresDB StatefulSet,
+		// if the configMap was not created at this step, NooBaaPostgresDB
+		// would fail to start.
+		if !util.KubeCheck(r.PostgresDBConf) {
```

Comment:
I think that Guy meant that you will check errors of the actual `KubeCreateSkipExisting` output

Reply:
@romayalon Existence of the configMap is required for the Postgres DB to start successfully. KubeCreateSkipExisting() returns false also in the case  "resource already exists", making examing this function return value inconclusive.  Makes more sense to test for the configMap existence. WDYT?

Reply:
you're right, the function name is misleading, see my comment above.

Reply:
@romayalon, thank you! Sounds good.

See https://github.com/baum/noobaa-operator/blob/39c5a4ea3cad97d10aef872aa3c86ca68cbc6966/pkg/system/reconciler.go#L270